### ISSUE TITLE
Error when recognize image

### ISSUE THREAD

**Issue by alexpong0630** [marcelklehr/recognize#34]
Hello,

I got this message after installation. My nextcloud is installed in a docker container.

`Classifier process output: sh: exec: line 1: /config/www/nextcloud/apps/recognize/bin/node-v14.9.0-linux-x64: not found`

But the file exists
```
root@b3b5321e5ad5:/root# ls /config/www/nextcloud/apps/recognize/bin/
node-v14.9.0-linux-arm64  node-v14.9.0-linux-armv7l  node-v14.9.0-linux-x64
```

**Comment by marcelklehr** [comment]
Are you using an alpine linux docker image?

**Comment by alexpong0630** [comment]
Yes. Is there any chance to make it work? I am looking for image classification for a long time.

**Comment by marcelklehr** [comment]
Alpine linux is missing glibc by default, I recommend switching to a different container image. Alternatively, you can have a look at how others solved this: https://github.com/marcelklehr/recognize/issues/16
